src/constants/tax.ts

```diff
@@ -0,0 +1,75 @@
+import { ChainId, Currency, Percent } from '@uniswap/sdk-core'
+
+import { ZERO_PERCENT } from './misc'
+
+interface TokenTaxMetadata {
+  buyFee?: Percent
+  sellFee?: Percent
+}
+
+const CHAIN_TOKEN_TAX_MAP: { [chainId in number]?: { [address in string]?: TokenTaxMetadata } } = {
+  [ChainId.MAINNET]: {
+    // BULLET
+    '0x8ef32a03784c8fd63bbf027251b9620865bd54b6': {
+      buyFee: new Percent(5, 100), // 5%
```

Comment:
Considering the duplication, it may be easier to understand if you just set some constants, eg

```
const ONE_PERCENT = new Percent(1, 100)
```